2024-10-22 11:29:59,141 - app_logger - ERROR - Error in sales_template_config: 'NoneType' object has no attribute 'sort'
2024-10-22 11:31:42,700 - app_logger - ERROR - Error in sales_template_config: 'dict' object has no attribute 'sort'
2024-10-22 11:41:00,260 - app_logger - ERROR - Error in model_reply: 'FastAPI' object has no attribute 'test_client'
2024-10-22 11:55:30,513 - app_logger - ERROR - Error in model_reply: 'module' object is not callable
2024-10-24 10:46:08,463 - app_logger - ERROR - Error in model_reply: '默认'
2024-10-28 16:53:27,550 - app_logger - ERROR - Error in slots_recognition: not enough values to unpack (expected 3, got 2)
2024-10-28 16:53:59,407 - app_logger - ERROR - Error in slots_recognition: string indices must be integers, not 'str'
2024-10-28 16:54:18,266 - app_logger - ERROR - Error in slots_recognition: string indices must be integers, not 'str'
2024-10-28 16:54:48,375 - app_logger - ERROR - Error in slots_recognition: string indices must be integers, not 'str'
2024-10-28 16:54:55,685 - app_logger - ERROR - Error in slots_recognition: string indices must be integers, not 'str'
2024-10-28 16:55:08,860 - app_logger - ERROR - Error in slots_recognition: string indices must be integers, not 'str'
2024-10-30 10:58:33,512 - app_logger - ERROR - Error in model_reply: 'NoneType' object has no attribute 'get'
2024-11-19 11:02:22,451 - app_logger - ERROR - Error in new_dialogue: GetSalesReplyStrict.__init__() missing 1 required positional argument: 'template_id'
2024-11-19 11:02:26,113 - app_logger - ERROR - Error in strict_reply: 'NoneType' object has no attribute 'intention_match_llm'
2024-11-19 11:07:11,261 - app_logger - ERROR - Error in new_dialogue: 'SetTemplate' object has no attribute 'template_intention'
2024-11-19 11:07:14,859 - app_logger - ERROR - Error in strict_reply: 'NoneType' object has no attribute 'intention_match_llm'
2024-11-19 11:11:34,044 - app_logger - ERROR - Error in strict_reply: '<generator object GetSalesReplyStrict.get_sales_reply.<locals>.<genexpr> at 0x7fc2e04bece0>'
2024-11-19 11:18:30,064 - app_logger - ERROR - Error in strict_reply: '<generator object GetSalesReplyStrict.get_sales_reply.<locals>.<genexpr> at 0x7fe3fa7aab20>'
2024-11-19 11:39:03,091 - app_logger - ERROR - Error in strict_reply: '{"intention": "问好"}'
2024-11-19 11:40:15,606 - app_logger - ERROR - Error in strict_reply: '{"intention": "问好"}'
2024-11-19 11:42:25,246 - app_logger - ERROR - Error in strict_reply: '{"intention": "问好"}'
2024-11-19 13:56:37,691 - app_logger - ERROR - Error in strict_reply: 'GetSalesReplyStrict' object has no attribute 'faq_reply'
2024-11-19 13:57:09,146 - app_logger - ERROR - Error in strict_reply: ''
2024-11-19 14:14:18,410 - app_logger - ERROR - Error in strict_reply: name 'date' is not defined
2024-11-19 14:17:24,654 - app_logger - ERROR - Error in slots_recognition: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
2024-11-19 14:19:06,847 - app_logger - ERROR - Error in strict_reply: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
2024-11-25 16:04:32,039 - app_logger - ERROR - Error in new_dialogue: <MilvusException: (code=2, message=Fail connecting to server on localhost:19530, illegal connection params or server unavailable)>
2024-11-25 16:08:31,155 - app_logger - ERROR - Error in new_dialogue: <MilvusException: (code=1100, message=Invalid collection name: 医美_美丽人生_美丽人生经典模版1. the first character of a collection name must be an underscore or letter: invalid parameter)>
2024-11-25 16:08:38,099 - app_logger - ERROR - Error in strict_reply: 'NoneType' object has no attribute 'intention_match_llm'
2024-11-25 16:08:38,104 - app_logger - ERROR - Error in slots_recognition: 'Doubao-pro-128k'
2024-11-25 16:13:11,620 - app_logger - ERROR - Error in new_dialogue: <MilvusException: (code=1100, message=Invalid collection name: 医美_美丽人生_美丽人生经典模版1. the first character of a collection name must be an underscore or letter: invalid parameter)>
2024-11-25 16:16:55,734 - app_logger - ERROR - Error in new_dialogue: <MilvusException: (code=1100, message=Invalid collection name: _医美_美丽人生_美丽人生经典模版1. collection name can only contain numbers, letters and underscores: invalid parameter)>
2024-11-25 16:31:01,504 - app_logger - ERROR - Error in intention_level: 'Doubao-pro-128k'
2024-11-25 16:41:51,248 - app_logger - ERROR - Error in new_dialogue: <SchemaNotReadyException: (code=1, message=Collection 'medical_beauty_classic_template1' not exist, or you can pass in schema to create one.)>
2024-11-25 16:44:23,153 - app_logger - ERROR - Error in intention_level: 'Doubao-pro-128k'
2024-11-25 16:44:34,074 - app_logger - ERROR - Error in new_dialogue: <MilvusException: (code=2, message=Fail connecting to server on 121.201.110.170:19530, illegal connection params or server unavailable)>
2024-11-25 17:01:04,592 - app_logger - ERROR - Error in new_dialogue: <SchemaNotReadyException: (code=1, message=Collection 'medical_beauty_classic_template1' not exist, or you can pass in schema to create one.)>
2024-11-25 17:46:15,433 - app_logger - ERROR - Error in strict_reply: 'Doubao-pro-128k'
2024-11-25 17:46:15,439 - app_logger - ERROR - Error in slots_recognition: 'Doubao-pro-128k'
2024-11-25 17:51:37,425 - app_logger - ERROR - Error in strict_reply: Milvus.search() got an unexpected keyword argument 'query_vector'
2024-11-25 17:53:54,947 - app_logger - ERROR - Error in strict_reply: Milvus.search() got an unexpected keyword argument 'top_k'
